lib/memcached.rb

```diff
@@ -1,14 +1,7 @@
-require 'memcached_rb'
-
 require 'memcached/client'
+require 'memcached/server'
+require 'memcached_rb'
```

Comment:
Gotta load the C extension after the Ruby code so it can pick the constants from it.